data/systemd/yggdrasil-sysuser.conf.in

```diff
@@ -0,0 +1,4 @@
+#Type Name            ID   GECOS                   Home directory             Shell
+u     @user@          -    "yggdrasil system user" @sharedstatedir@/yggdrasil -
+u     @worker_user@   -    "yggdrasil worker user" -                          -
+m     @user@          @worker_user@
```

Comment:
It is correct, but it looks strange. This adds user to supplementary group `@worker_user@` (replaced by default by `yggdrasil-worker`).

I think that we should move echo worker to its own RPM package, and it will solve some issues. We can do this in another PR.